history/states/309-Dutch SA.txt

```diff
@@ -1,31 +1,31 @@
-
-state={
-	id=309
-	name="STATE_309"
-	manpower = 152000
-
-	resources={
-		aluminium=55 # was: 104
-	}	
-	
-	state_category = rural
-
-	history={
-		add_core_of = SUR
-		owner = HOL
-		buildings = {
-			infrastructure = 3
-			industrial_complex = 1
-			12828 = {
-				naval_base = 1
-			}
-		}
-		victory_points = {
-			10906 1 
-		}		
-	}
-
-	provinces={
-		1928 10906 12828 
-	}
-}
+
+state={
+	id=309
+	name="STATE_309"
+	manpower = 15110

```

Comment:
It seems you missed a number. instead of 150k the new mapower really 15k?

Reply:
I might have commited a little mistake here. I had to give the new states some manpower, maybe I did a mistake along the way.